internal/server/routes.go

```diff
@@ -57,6 +57,7 @@ func (s *Server) GenerateRoutes(promRegistry prometheus.Registerer) Routes {
 	apiGroup := router.Group("/",
 		metrics.Middleware(promRegistry),
 		DatabaseMiddleware(a.server.db), // must be after TimeoutMiddleware to time out db queries.
+		DestinationMiddleware(),
```

Comment:
I think we'll probably need to put this after the `AuthenticationMiddleware` so that we can authorize the request to update the connector last seen.

Reply:
Good call. For this draft, I elected to make a minimal set of changes. The auth check will definitely be in the final version